dask/dataframe/tests/test_dataframe.py

```diff
@@ -1712,16 +1713,17 @@ def test_apply():
               df.apply(lambda xy: xy[0] + xy[1], axis='columns'))
 
     # inference
-    assert_eq(ddf.apply(lambda xy: xy[0] + xy[1], axis=1),
+    assert_eq(ddf.apply(lambda xy: xy[0] + xy[1], axis=1, meta=(None, int)),
               df.apply(lambda xy: xy[0] + xy[1], axis=1))
-    assert_eq(ddf.apply(lambda xy: xy, axis=1),
+    assert_eq(ddf.apply(lambda xy: xy, axis=1, meta={'x': int, 'y': int}),
               df.apply(lambda xy: xy, axis=1))
 
     # specify meta
     func = lambda x: pd.Series([x, x])
     assert_eq(ddf.x.apply(func, meta=[(0, int), (1, int)]), df.x.apply(func))
     # inference
-    assert_eq(ddf.x.apply(func), df.x.apply(func))
+    with pytest.warns(UserWarning):
```

Comment:
Are we really expected a warning in this kind of call? ISTM the test will start failing if the warning disappears... If the intent is to silence warnings without expecting them I'd rather have a specific context manager for that.